src/nodejs/hooks/sealights.go

```diff
@@ -346,62 +402,140 @@ func (sl *SealightsHook) injectSealights(stager *libbuildpack.Stager) error {
 	}
 }
 
-func containsSealightsService(key string, services interface{}, query string) bool {
-	var serviceName string
+func (sl *SealightsHook) parseVcapServices() {
 
-	if strings.Contains(key, query) {
-		return true
+	if sl.initialized {
+		sl.Log.Debug("already initialized. config won`t be parsed")
```

Comment:
Please remove the debug logs

Reply:
This log not shown by default
user should configure Debug mode to see that